core/ui/src/main/kotlin/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -118,6 +118,7 @@ fun NewsResourceCardExpanded(
         // Pass null for action to only override the label and not the actual action.
         modifier = modifier
             .semantics {
+                contentDescription = clickActionLabel
```

Comment:
Let me know if you prefer to use `userNewsResource.title` or if another workaround might be better (adding a header semantic somewhere in the card maybe?)